R/collectors.R

```diff
@@ -216,6 +217,9 @@ guess_parser <- function(x, locale = default_locale()) {
 #'
 #' # parse_factor generates a warning & problems
 #' x2 <- parse_factor(x, levels)
+#'
+#' # Using an argument of `NULL` will generate levels based on values of `x`
+#' x2 <- parse_factor(x, levels = NULL)
```

Comment:
I guess it is implementation dependant based on std::map, how should we order them, just sorted alphabetically?